ui/components/app/modals/export-private-key-modal/export-private-key-modal.test.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { fireEvent } from '@testing-library/react';
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import { renderWithProvider } from '../../../../../test/lib/render-helpers';
+import mockState from '../../../../../test/data/mock-state.json';
+import * as actions from '../../../../store/actions';
+import ExportPrivateKeyModal from '.';
+
+jest.mock('../../../../store/actions.ts', () => ({
+  ...jest.requireActual('../../../../store/actions.ts'),
+  exportAccount: jest.fn().mockReturnValue(jest.fn().mockResolvedValueOnce()),
+  hideWarning: () => jest.fn(),
+  showModal: () => jest.fn(),
+  hideModal: () => jest.fn(),
+  clearAccountDetails: () => jest.fn(),
+}));
+
+describe('Export PrivateKey Modal', () => {
+  const password = 'a-password';
+
+  const privKeyModalState = {
+    ...mockState,
+    appState: {
+      ...mockState.appState,
+      accountDetail: {
+        privateKey: '0xPrivKey',
+      },
+    },
+  };
+  const mockStore = configureMockStore([thunk])(privKeyModalState);
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should match snapshot', () => {
+    const { container } = renderWithProvider(
+      <ExportPrivateKeyModal />,
+      mockStore,
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should disable confirm button by default', () => {
```

Comment:
We can skip this test as well as it is covered in 
https://github.com/MetaMask/metamask-extension/blob/develop/ui/components/app/modal/modal.component.test.js#L85
wdyt?